frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -635,6 +671,14 @@ export class TileViewPage extends React.Component {
     const updatedFilters = updateActiveFilters(activeFilters, filterType, id, value);
 
     this.updateMountedState(this.getUpdatedState(categories, selectedCategoryId, updatedFilters));
+
+    if (this.props.storeFilterKey && updatedFilters) {
+      const storeFilters = {};
+      _.each(this.props.filterRetentionPreference, (filterGroup) => {
+        updatedFilters[filterGroup] && (storeFilters[filterGroup] = updatedFilters[filterGroup]);
```

Comment:
Let's avoid setting values as the right-hand operation of a truthy check...
```suggestion
        if (updatedFilters[filterGroup]) {
          storeFilters[filterGroup] = updatedFilters[filterGroup];
        }
```

Reply:
updated